kolibri/deployment/default/settings/base.py

```diff
@@ -195,5 +195,10 @@
 
 AUTH_USER_MODEL = 'kolibriauth.DeviceOwner'
 
+AUTH_ANONYMOUS_USER_MODEL = "kolibri.auth.models.KolibriAnonymousUser"
```

Comment:
Why create a new setting at all? If it's only the configuration of DRF that uses this, then just hard-code that. As soon as there is a second use, then I guess we should consider how to make it available.